.vscode/launch.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Hm, this brings up an interesting point. I would normally vouch for .gitignoring the `.vscode` folder entirely, but clearly this file makes sense to keep in there. The same goes for the settings file that I have which states the unusual tab size of 4